molecule/command/lint.py

```diff
@@ -71,18 +73,21 @@ def execute(self):
         :return: None
         """
         self.print_info()
-        linters = [
-            l
-            for l in [
-                self._config.lint,
-                self._config.verifier.lint,
-                self._config.provisioner.lint,
-            ]
-            if l
-        ]
-
-        for l in linters:
-            l.execute()
+
+        # v3 migration code:
+        cmd = self._config.lint
+        if cmd == 'yamllint':
+            LOG.warning(
+                "Deprecated linter command found, migrate to newer syntax using <url>"
+            )
+            cmd = 'yamllint .'
+
+        try:
+            # import pdb; pdb.set_trace()
+            LOG.info("Executing: %s" % cmd)
+            util.run(cmd, shell=True, universal_newlines=True, check=True)
```

Comment:
maybe worth to support `env` here

Reply:
Not in this PR. In fact this will reuse env from parent process. I want to avoid extra logic arond env because if needed it means that user would not be able to run the linter w/o molecule.